pkg/image/export/main.go

```diff
@@ -267,35 +267,46 @@ func getK3sUpgradeImages(rancherVersion string, k3sData map[string]interface{})
 			continue
 		}
 
-		if rancherVersion != "dev" {
-			maxVersion, _ := releaseMap["maxChannelServerVersion"].(string)
-			maxVersion = strings.TrimPrefix(maxVersion, "v")
-			if maxVersion == "" {
-				continue
-			}
-			minVersion, _ := releaseMap["minChannelServerVersion"].(string)
-			minVersion = strings.Trim(minVersion, "v")
-			if minVersion == "" {
-				continue
-			}
+		maxVersion, _ := releaseMap["maxChannelServerVersion"].(string)
+		maxVersion = strings.TrimPrefix(maxVersion, "v")
+		if maxVersion == "" {
+			continue
+		}
+		minVersion, _ := releaseMap["minChannelServerVersion"].(string)
+		minVersion = strings.Trim(minVersion, "v")
+		if minVersion == "" {
+			continue
+		}
 
-			versionGTMin, err := k3sbasedupgrade.IsNewerVersion(minVersion, rancherVersion)
-			if err != nil {
-				continue
-			}
-			if rancherVersion != minVersion && !versionGTMin {
-				// rancher version not equal to or greater than minimum supported rancher version
-				continue
-			}
+		var versionGTMin bool
+		var err error
+		if rancherVersion == kd.RancherVersionDev {
```

Comment:
So `rancherVersion` would never be `dev` right ? It always gets translated to rancherVersionDev

Reply:
Correct, as it is right now this function only gets used in one place where dev and master are translated.

Reply:
ok cool